### PR TITLE
Create multi select dropdown component

### LINKED ISSUE #137: [Optional] Add search functionality in dropdown
It can take a while until you find the county

image

### LINKED ISSUE #144: Create multi select dropdown component
Needed for County selection

image

### PR DESCRIPTION
**Pull request by bogdanmosica**
Fixes #144
Fixes #137

using: https://github.com/hc-oss/react-multi-select-component

### CODE REVIEW

File: components/Form/DropdownMultiSelect.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  const { t } = useTranslation('common')
+
+  const valueRenderer = (selected: typeof options) => {
+    if (!selected.length) {
+      return t('services.county.placeholder')
+    }
+
+    if (selected.length === 1) return selected[0].label
+    // TODO add translation
+    return `${selected.length} judete selectate`
```
Comment (cleferman) [inline components/Form/DropdownMultiSelect.tsx:27]: don't forget about the TODO :)

File: forms/TransportGoodsForm.tsx [hunk 2]
```diff
@@ -153,44 +188,64 @@ export const TransportGoodsForm = ({ onSubmit }: ITransportGoodsFormProps) => {
...
                 errors={errors.transportCounty}
                 placeholder={t('services.county.placeholder')}
               >
                 {countiesOptions}
               </Dropdown>
             </Radio>
+            {/* 
+                // TODO Remove this after testing DropdownMultiSelect
+              */}
+            {/* <DropdownMultiSelect
```
Comment (cleferman) [inline forms/TransportGoodsForm.tsx:212]: shouldn't this be the other way around? The Radio should include the control

**Review by cleferman — CHANGES_REQUESTED** [review]

File: forms/TransportGoodsForm.tsx (hunk 2 above)
Comment (cipick) [reply forms/TransportGoodsForm.tsx:212]: no, the radio will contain the choices for `type` and if `type` is County

**Review by cipick — APPROVED** [review]
awesome, thanks a lot

**Pull request merged**